File: JavaProjects/blog-frontend/src/types/Comment.ts
import { User } from "./User";
import { v4 as uuid } from "uuid";
import { useUser } from "../stores/user";

export interface Comment {
  id: string;
  articleId: string;
  userId: number;
  commentContent: string;
  commentTime: number;
  user: User;
}

export function commentCreator(user: User) {
  return {  }
}

File: JavaProjects/BlogSystem/src/main/java/com/bricks/blogsystem/entities/Comment.java
package com.bricks.blogsystem.entities;

import lombok.Data;
import org.springframework.stereotype.Component;

import java.sql.Date;

@Data
@Component
public class Comment {
    private Long id;
    private Long articleId;
    private Long userId;
    private String commentContent;
    private Date commentTime;
    private Long parentCommentId;
}